src/components/Checklist.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import _ from 'lodash'
```

Comment:
This is being used here, but is not in the package.json file. However, before you add it, see comments below.